Directory.Build.props

```diff
@@ -13,4 +13,15 @@
     <RootOutDir>$(ReactNativeWindowsDir)\target</RootOutDir>
   </PropertyGroup>
 
-</Project>
\ No newline at end of file
+  <PropertyGroup Label="NuGet">
+    <MSBuildProjectExtensionsPath>$(RootIntDir)\ProjectExtensions\$(MSBuildProjectName)\</MSBuildProjectExtensionsPath>
```

Comment:
MSBuildProjectExtensionPath is defined in Microsoft.Common.props
Why do we need to define it here?

Reply:
Yes, in order to have the restore-generated meta files under the intermediate output directory.
This is a safe change.